projects/00-trivia/README.pt-BR.md

```diff
@@ -49,33 +49,33 @@ Reflita e depois enumere os objetivos que quer alcançar e aplique no seu
 projeto. Pense nisso para decidir sua estratégia de trabalho individual e em
 equipe.
 
-### UX
-
-* [ ] Desenhar a aplicação pensando e entendendo o usuário.
-* [ ] Criar protótipos para obter _feedback_ e implementar.
-
 ### HTML e CSS
 
-* [ ] Entender e reconhecer a importância do HTML semântico.
-* [ ] Identificar e entender tipos de seletores em CSS.
-* [ ] Construir uma aplicação respeitando o protótipo.
+* [ ] [Uso de HTML semântico.](https://developer.mozilla.org/pt-BR/docs/Glossario/Semantica#Sem%C3%A2ntica_em_HTML)
+* [ ] Uso de seletores de CSS.
+* [ ] Construa sua aplicação respeitando o design feito (layout).
 
 ### DOM
 
-* [ ] Entender e reconhecer os seletores do DOM.
-* [ ] Manejar os eventos do DOM.
-* [ ] Manipular dinamicamente o DOM.
+* [ ] Uso de seletores do DOM.
+* [ ] Gestão dos eventos do DOM.
+* [ ] [Manipulação dinâmica do DOM](https://developer.mozilla.org/pt-BR/docs/DOM/Referencia_do_DOM/Introdu%C3%A7%C3%A3o)
 
 ### JavaScript
 
-* [ ] Manipular strings.
-* [ ] Entender o uso de condicionais.
-* [ ] Utilizar funções.
-* [ ] Conhecer a diferença entre os distintos tipos de variáveis.
+* [ ] Manipulação de strings.
```

Comment:
In a change proposed by @luucamay a few months ago and recently merged to `master` the [JavaScript learning objectives](https://github.com/Laboratoria/bootcamp/tree/master/projects/00-trivia#javascript) have been updated slighly:

1. _Manipular strings_ was changed to _Concatenación de strings_.
2. _Tipos de datos_ was added.
3. The ordering was changed.

Can you please update these so that we can proceed with the merge? Alternatively I can also offer to make the changes directly :wink:

Reply:
@lupomontero Listo! Creo que he corrigido!

Reply:
Gracias @moniyama :1st_place_medal: